commit a1773ab14b6cea952cfa34f030c1948b3b3eb565 (refs/changes/63/163/1)
Author: Johanan Lai <lai.johanan@gmail.com>
Date:   2016-07-26T17:16:14-07:00 (3 years, 2 months ago)
    
    ADMINUI-2326 Clicking "New Limit" twice while provisioning a limit causes the form to change from New Limit -> Update Limit for undefined

diff --git a/www/js/components/pages/user/limits/main.jsx b/www/js/components/pages/user/limits/main.jsx
index 06c1b3ef..cbeb5ea4 100644
--- a/www/js/components/pages/user/limits/main.jsx
+++ b/www/js/components/pages/user/limits/main.jsx
@@ -90,7 +90,7 @@ var ProvisioningLimits = React.createClass({
                 <h3>Provisioning Limits
                     {adminui.user.role('operators') &&
                         <div className="actions">
-                            <button onClick={this.showNewLimitForm} className="btn btn-sm btn-info"><i className="fa fa-plus"></i> New Limit</button>
+                            <button onClick={this.showNewLimitForm} className="btn btn-sm btn-info" disabled={this.state.form ? 'disabled' : ''}><i className="fa fa-plus"></i> New Limit</button>
                         </div>}
                 </h3>
                 <ProvisioningLimitsList
